compileopts/target.go

```diff
@@ -253,7 +254,7 @@ func defaultTarget(goos, goarch, triple string) (*TargetSpec, error) {
 	}
 	if goarch != runtime.GOARCH {
 		// Some educated guesses as to how to invoke helper programs.
-		spec.GDB = "gdb-multiarch"
+		spec.GDB = []string{"gdb-multiarch", "arm-none-eabi-gdb"}
```

Comment:
This change is not correct. This code is for regular desktop usage, where `gdb-multiarch` is usually the correct name and `arm-none-eabi-gdb` never is.

Reply:
arm-none-eabi-gdb has been removed.